docs/DeveloperGuide.adoc

```diff
@@ -25,10 +25,10 @@
 . Rename the project if you want. Click `Next`
 . Ensure that your `\src` and `\test\java` folders are checked. Keep clicking `Next`
 . Click `Finish`
-. Add JUnit 4 to classpath
+. Add JUnit 5 to classpath
 .. Open any test file in `\test\java` and place your cursor over any `@Test` highlighted in red
 .. Press kbd:[ALT + ENTER] and select `Add 'JUnit4' to classpath`
-.. Select `junit:junit:4.12` and click `OK` (if `4.12` is not available, choose any `4.x` that is later than `4.12`)
+.. Select `junit:junit:5.40` and click `OK` (if `5.40` is not available, choose any `5.4x` that is later than `5.40`)
```

Comment:
`junit:junit:5.40` should not be possible since JUnit5 is using a different artifact/group name on maven.

Maybe run through the instructions in IntelliJ? It says `org.junit.jupiter:junit-jupiter-api:RELEASE` on my side.

Reply:
I was actually thinking that they should hit the refresh button and select 5.4.x from the dropdown since RELEASE might point to a release that breaks later on.

Reply:
> I was actually thinking that they should hit the refresh button and select 5.4.x from the dropdown since RELEASE might point to a release that breaks later on.

Oh Okay I can see the list of versions now after waiting (a long time for some reason).

Reply:
Yeah on different versions of of IDEA the list fetch varies in size as well. Kind of annoying, should we consider putting a screenshot in? Messing this step up could result in a fair bit of trouble for students.

Reply:
> Kind of annoying, should we consider putting a screenshot in?

ok.

Another idea: how about just asking students to replace `org.junit.jupiter:junit-jupiter-api:RELEASE` with `org.junit.jupiter:junit-jupiter-api:5.4.0` and then pressing OK directly? That way we won't need to depend on IntelliJ getting the list correct, and it's just a copy and paste.